lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +300,35 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	pageSize, pageToken, err := obsidian.GetPageSizeAndTokenParams(c)
+	if err != nil {
+		return err
+	}
 
-	ents, _, err := configurator.LoadAllEntitiesOfType(
+	ents, nextPageToken, err := configurator.LoadAllEntitiesOfType(
 		nid, lte.CellularEnodebEntityType,
-		configurator.EntityLoadCriteria{LoadMetadata: true, LoadConfig: true, LoadAssocsToThis: true},
+		configurator.EntityLoadCriteria{
+			LoadMetadata:     true,
+			LoadConfig:       true,
+			LoadAssocsToThis: true,
+			PageSize:         uint32(pageSize),
+			PageToken:        pageToken},
 		serdes.Entity,
 	)
 	if err != nil {
 		return obsidian.HttpError(err, http.StatusInternalServerError)
 	}
 
-	ret := make(map[string]*lte_models.Enodeb, len(ents))
+	enodebs := make(map[string]*lte_models.Enodeb, len(ents))
 	for _, ent := range ents {
-		ret[ent.Key] = (&lte_models.Enodeb{}).FromBackendModels(ent)
+		enodebs[ent.Key] = (&lte_models.Enodeb{}).FromBackendModels(ent)
```

Comment:
From offline: it looks like we use map of key->ptr everywhere. If that's the case, then it's fine to leave the "xnullable: true" in the IDL defn, just add a comment to that IDL line with why we have it there